barcelona/scraping.py

```diff
@@ -0,0 +1,41 @@
+import urllib.request
+import datetime
+import time
+import json
+
+# Create header of csv file with column names
+# Only has to be ran once
+with open("barcelona/test_scrape.csv", mode="w+") as f:
+    f.write('time;id;type;latitude;longitude;streetName;streetNumber;altitude;slots;bikes;nearbyStations;status')
+    f.write("\n")
+
+# Define function to get json response
+def get_response(url):
+    resp = urllib.request.urlopen(url)
+    data = resp.read()
+    json_dt = json.loads(data)
+    return(json_dt)
+
```

Comment:
with `requests` library this could become
```
def get_response(url):
    response = requests.get(url)
    
    return response.json()
```
How do you like it?